lib/llvmopencl/linker.cpp

```diff
@@ -458,7 +458,9 @@ static void replaceIntrinsics(llvm::Module *Program, const llvm::Module *Lib,
     }
   }
 
-  for (auto [Intrin, Repl] : EraseMap) {
+  for (auto It : EraseMap) {
```

Comment:
Isn't C++17 OK? We support the oldest C++ standard supported by the oldest LLVM we support. I think LLVM moved to C++17 a while ago.

Reply:
LLVM 14&15 use C++14, the move only happened with LLVM 16.